doc/source/ray-air/check-ingest.rst

```diff
@@ -83,20 +87,31 @@ Enabling Streaming Ingest
 
     You should use bulk ingest when:
 
-     * you have enough memory to fit data blocks in cluster object store;
-     * your preprocessing step is expensive per each epoch; and
-     * you want best performance when both or either the above conditions are met.
+    * you have enough memory to fit data blocks in cluster object store; or
+    * your preprocessing transform is expensive to recompute on each epoch
 
 .. tabbed:: Streaming Ingest (experimental)
 
-    In streaming ingest mode, :py:func:`~ray.air.session.get_dataset_shard` returns a :py:class:`~ray.data.dataset_pipeline.DatasetPipeline` pipeline that
-    can be used to read data in a streaming way.
-    To enable streaming ingest, set ``use_stream_api=True`` in the dataset config.
+    In streaming ingest mode, instead of loading the entire dataset into the
+    Ray object store at once, AIR will load a fraction of the dataset at a
+    time. This can be desirable when the dataset is very large, and caching it
+    all at once would cause expensive disk spilling. The downside is that the
+    dataset will have to be preprocessed on each epoch, which can add overhead if
+    the computation is expensive.
```

Comment:
Sorry, do you have a change to request?

I think that's partly right, but the GPUs won't necessarily be idle for bulk ingest training. They should only be idle on the first epoch. If you're doing multiple passes over the same data, bulk ingest is generally more likely to yield higher GPU utilization than streaming because the data is cached.

Reply:
Yeah agreed it depends on a multitude of factors.

The suggestion is more just to clarify that computation is overlapped so the downside of preprocessing per epoch may not be that egregious.

Reply:
Got it, will do!